### PR TITLE
Use SeedServicesApiClient for Scheduler and Service rating services

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

File: seed_services_client/scheduler.py [hunk 1]
```diff
@@ -3,74 +3,42 @@
...
-
+class SchedulerApiClient(SeedServicesApiClient):
     """
-    Client for Scheduler API.
-
-    :param str api_token:
+    Client for Scheduler Service.
 
-        An API Token.
+    :param str auth_token:
```
Comment (erikh360) [inline seed_services_client/scheduler.py]: Shouldn't this be in the same place as the `__init__` function? The `seed_services.py` file.

File: seed_services_client/service_rating.py [hunk 2]
```diff
@@ -11,18 +11,10 @@ class ServiceRatingApiClient(object):
     :param str api_url:
         The full URL of the API.
 
-    """
-
-    def __init__(self, auth_token, api_url, session=None, session_http=None):
-        if session is None:
-            session = JSONServiceClient(
-                url=api_url, headers={'Authorization': 'Token ' + auth_token})
+    :param JSONServiceClient session:
```
Comment (erikh360) [inline seed_services_client/service_rating.py]: Same as above?

**Review by erikh360 — COMMENTED** [review]
Just the one question in two places, otherwise it's looking good.

File: seed_services_client/scheduler.py (hunk 1 above)
Comment (KaitCrawford) [reply seed_services_client/scheduler.py]: The comments are already there if that's what you mean
https://github.com/praekeltfoundation/seed-services-client/blob/develop/seed_services_client/seed_services.py#L23
I can remove them from here if you think the redundancy is confusing?

Comment (erikh360) [reply seed_services_client/scheduler.py]: The problem is if we make any changes to the parameters of the `__init__` function we will have to update comments in multiple places.

**Review by alexmuller — APPROVED** [review]
Looks right to me, the test response more closely resembles how the service actually operates 👍

**Pull request merged**